SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2266,6 +2268,26 @@ void ActorSystem::UpdateInterestComponent(AActor* Actor)
 		NetDriver->InterestFactory->CreateInterestUpdate(Actor, NetDriver->ClassInfoManager->GetOrCreateClassInfoByObject(Actor), EntityId);
 
 	NetDriver->Connection->SendComponentUpdate(EntityId, &Update);
+
+	if (Actor->IsA(APlayerController::StaticClass()) && GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)
+	{
+		Worker_CommandRequest CommandRequest{};
+		const bool bRequestValid = NetDriver->InterestFactory->CreateClientInterestDiff(Actor, CommandRequest, bOverwriteInterest);
+
+		if (bRequestValid)
+		{
+			const Worker_EntityId SystemEntityId =
+				Cast<USpatialNetConnection>(Actor->GetNetConnection())->ConnectionClientWorkerSystemEntityId;
+
+			NetDriver->Connection->SendCommandRequest(SystemEntityId, &CommandRequest, RETRY_MAX_TIMES, {});
+
+			UE_LOG(LogTemp, Warning, TEXT("Interest diff: worker entity id %lld"), SystemEntityId);
+		}
+		else
+		{
+			UE_LOG(LogTemp, Warning, TEXT("Interest diff: refresh request but no diff detected (%s)"), *Actor->GetName());
```

Comment:
big boy error UNLESS we're doing an overwrite (although for that bRequestValid might not ever return true? idk)